models/pancakeswap/bnb/pancakeswap_bnb_schema.yml

```diff
@@ -0,0 +1,96 @@
+version: 2
+
+models:
+  - name: pancakeswap_v2_bnb_trades
+    meta:
+      blockchain: bnb
+      sector: dex
+      project: pancakeswap_v2
+      contributors: hosuke
+    config:
+      tags: ['bnb','pancakeswap_v2','trades', 'pancakeswap', 'dex']
+    description: >
+        Pancake swap V2 contract trades on BNB smart chain
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - block_date
+            - blockchain
+            - project
+            - version
+            - tx_hash
+            - evt_index
+            - trace_address
```

Comment:
@jeff-dude I added an unique tests here for pancakeswap v2 trades, I will retest with a strengthen join in sql

Reply:
no, these are correct, we'll apply a unique test to prices model soon (hopefully tomorrow)